drivers/i3c/i3c_dw.c

```diff
@@ -2357,13 +2406,6 @@ static int dw_i3c_init(const struct device *dev)
 	__ASSERT_NO_MSG((IS_ENABLED(CONFIG_I3C_TARGET) && ctrl_config->is_secondary) ||
 			(IS_ENABLED(CONFIG_I3C_CONTROLLER) && !ctrl_config->is_secondary));
 
-	ret = init_scl_timing(dev);
-	if (ret != 0) {
-		return ret;
-	}
-
-	enable_interrupts(dev);
-
```

Comment:
This needs to be added back wtih it calling that `dw_i3c_init_scl_timing` rather than the old name